src/CommonEvents.php

```diff
@@ -105,4 +106,45 @@ public function canDispatchVirtualHostOrFail(QuestionHelper $helper, InputInterf
             }
         }
     }
+
+    /**
+     * @throws CannotHandleEventException
+     */
+    public function dispatchImage(Service $service, QuestionHelper $helper, InputInterface $input, OutputInterface $output): void
+    {
+        $this->canDispatchImageOrFail($helper, $input, $output);
+
+        Hermes::dispatchJson(self::NEW_IMAGE, $service);
+    }
+
+    /**
+     * @throws CannotHandleEventException
+     */
+    public function canDispatchImageOrFail(QuestionHelper $helper, InputInterface $input, OutputInterface $output): void
+    {
+        $canHandle = Hermes::canHandleEvent(self::NEW_IMAGE);
+
+        if (!$canHandle) {
+            $output->writeln('<error>Heads up!</error>');
+            $output->writeln('It seems that Aenthill does not know how to handle the creation of a new image. You need to install a dedicated Aent for this.');
+            $output->writeln('Most of the time, you want to put the instructions in a Dockerfile. We have a pretty good Aent for this: <info>theaentmachine/aent-dockerfile</info>.');
+            $question = new Question('Do you want me to add this Aent for you? (y/n) ', 'y');
+            $question->setValidator(function (string $value) {
```

Comment:
We should refactor this to use the new yes/no question helper (and in the method above too)